recipes/kim_edn/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "kim_edn" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bab88a0a890598577a5dabe0ff19ed573ef7a0be547e5a47b15deeb4ec711db5
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
```

Comment:
I think this recipe can be `noarch`, https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages. In that case, please remove the `skip` line, and add `noarch: python` instead.